astropy/utils/masked/core.py

```diff
@@ -1428,3 +1429,33 @@ def __repr__(self):
         out0 = cls_name + "(" + rest
         extra_space = (len(cls_name) - len(prefix)) * " "
         return "\n".join([out0] + [extra_space + o for o in out[1:]])
+
+
+_base_class_qualnames = [
+    "astropy.units.quantity.Quantity",
+    "astropy.coordinates.angles.core.Angle",
+    "astropy.coordinates.angles.core.Latitude",
+    "astropy.coordinates.angles.core.Longitude",
+]
+
+
+def __getattr__(key):
+    """Make commonly used Masked subclasses importable for ASDF support.
```

Comment:
I don't think this docstring would be rendered in API doc because it is hidden inside a dunder call. Do you want it to?

Reply:
No, I don't want it in the docs. This comment is really more for developers.